pages/help-center/errors/serialization.mdx

```diff
@@ -1,53 +1,101 @@
 ---
-title: Serialization errors
-description: Learn about serialization errors as well as troubleshooting best practices for serialization errors in our all-in-one documentation page at Memgraph. 
+title: Conflicting transaction and serialization error
+description: Learn about conflicting transaction and serialization errors as well as troubleshooting best practices for serialization errors on our all-in-one documentation page at Memgraph. 
 ---
 
 import { Callout } from 'nextra/components'
 
-# Serialization error
+# Conflicting transaction and serialization error
 
 <Callout type="info">
 If you are encountering issues with serialization errors, we're here to help. Join the conversation on <a href="https://www.discord.gg/memgraph" target="_blank">Discord</a> for support.
 </Callout>
 
-## Error
+## Error messages
 
 1. [Unable to commit due to serialization error.](#error-1)
+2. [Cannot resolve conflicting transactions.](#error-1)
```

Comment:
This part should be removed now?

Reply:
I would leave it for the time being, not 100% sure that there is no log from serialization error in QM for example. So let's keep it for now.

Reply:
But these two lines are links to the non-existent sections.

Reply:
Aha, let me check.

Reply:
Done